tests/test_cmdline.py

```diff
@@ -21,6 +21,27 @@
 logger = logging.getLogger(__name__)
 
 
+def make_settings_dict(**kwargs):
+    """Create an expected version of Settings.dict(), with customizations.
+
+    Return a copy of DEFAULTS, with the given customizations applied.
+    """
```

Comment:
I'd like "json" mentioned somewhere in the name or docstring here, to emphasize that this creates a settings dict of the JSON flavor, as opposed to the equivalent function in `test_settings`. Also, the `DEFAULTS` in the above docstring does not really make sense now.

Reply:
Done in [b4398da](https://github.com/tweag/FawltyDeps/pull/261/commits/b4398da4fa25c129c7fd37b78c1f01a6e43b0340).